inference-engine/tests/ngraph_functions/src/activation.cpp

```diff
@@ -93,6 +93,8 @@ std::shared_ptr<ngraph::Node> makeActivation(const ngraph::Output<Node> &in,
         }
         case ngraph::helpers::ActivationTypes::Mish:
             return std::make_shared<ngraph::op::v4::Mish>(in);
+        case ngraph::helpers::ActivationTypes::HSwish:
+            return std::make_shared<ngraph::op::v4::HSwish>(in);
```

Comment:
Please rebase your branch on latest master. I merged https://github.com/openvinotoolkit/openvino/pull/1861/files to the master branch recently

Reply:
Thank you, I rebased this branch on master